doc/examples/introduction.ipynb

```diff
@@ -25,6 +25,22 @@
     "import xarray as xr"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`cf_xarray` works best when `xarray` keeps attributes by default.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xr.set_options(keep_attrs=True)"
```

Comment:
Looks like we've been running the intro notebook without `xr.set_options(keep_attrs=True)`.
I don't think the issue without it was triggered by this PR, but I might be wrong.

Anyways, I think we should keep attributes and maybe re-discuss #77?